### PR TITLE
Build both python and python3 plugins.

### PR DESCRIPTION
**Pull request by igorpeshansky**
Bump minor version to 6.3.0.

http://b/168508440

### CODE REVIEW

File: rpm/stackdriver-agent.spec [hunk 1]
```diff
@@ -110,11 +108,8 @@ Source202: stackdriver.sysconfig
 BuildRequires: perl(ExtUtils::MakeMaker)
 BuildRequires: perl(ExtUtils::Embed)
 %endif
-%if ! %{has_python36}
```
Comment (qingling128) [inline rpm/stackdriver-agent.spec:113]: Do we need to change anything in the sles 12 and 15 dockerfile (https://github.com/Stackdriver/agent-packaging/blob/1f1ee6b739fc50fbe27d1bfc2214f6386515b70f/build/x86_64_sles12/Dockerfile#L29) to add Python 3 as well? Or do they have python2-devel and python3-devel already?

Comment (igorpeshansky) [reply rpm/stackdriver-agent.spec:113]: Great catch. Yes, those and the Amazon `Dockerfile` also needed to be modified. However, they are currently broken at HEAD, so we first need to get them working again. Sent #132 — once that's merged, I'll update these.

Comment (igorpeshansky) [reply rpm/stackdriver-agent.spec]: Restored the build containers and rebased this PR.

**Review by qingling128 — APPROVED** [review]
LGTM

File: build/x86_64_bionic/Dockerfile [hunk 2]
```diff
@@ -27,6 +27,7 @@ RUN apt-get -y update \
         lsb-release \
         make \
         pkg-config \
-        python-dev \
+        python2-dev \
```
Comment (igorpeshansky) [inline build/x86_64_bionic/Dockerfile:30]: Rats, I thought I tested them all, but I forgot bionic. 😦 It didn't have python2-dev, only python-dev. Sent #136.

Comment (igorpeshansky) [inline build/x86_64_bionic/Dockerfile:30]: Rats, I thought I tested them all, but I forgot bionic. 😦 It didn't have python2-dev, only python-dev. Sent #136.

File: rpm/stackdriver-agent.spec (hunk 1 above)
Comment (igorpeshansky) [reply rpm/stackdriver-agent.spec]: Also Debian stretch, apparently. 😦 Sent #137.

**Pull request merged**